tests/integration/test_charm.py

```diff
@@ -28,16 +28,17 @@
 @pytest_asyncio.fixture(name="deploy", scope="module")
 async def deploy(ops_test: OpsTest):
     """The app is up and running."""
-    charm = await ops_test.build_charm(".")
-    resources = {"temporal-ui-image": METADATA["containers"]["temporal-ui"]["upstream-source"]}
-
     # Deploy temporal server, temporal admin and postgresql charms.
-    await ops_test.model.deploy(charm, resources=resources, application_name=APP_NAME)
     await ops_test.model.deploy(APP_NAME_SERVER, channel="stable")
     await ops_test.model.deploy(APP_NAME_ADMIN, channel="stable")
     await ops_test.model.deploy("postgresql-k8s", channel="edge", trust=True)
```

Comment:
Out of curiosity, what does edge version of postgresql-k8s charm allow for that the stable does not?

Reply:
This was a remnant of the older integration tests when we were using the edge channel. I've updated the integration test to use the 14/stable channel.